test-e2e/config.js

```diff
@@ -0,0 +1,33 @@
+// These config are used by tests and devServer.
```

Comment:
should this file be `.ts` ?

Reply:
I need to tackle this in a future PR because codeceptjs requires this config, and it is running as Javascript.